scripts/coverity_misra.config

```diff
@@ -26,6 +26,10 @@
             deviation: "Rule 2.5",
             reason: "Allow unused macros. Library headers may define macros intended for the application's use, but not used by a specific file."
         },
+        {
+            deviation: "Rule 11.5",
+            reason: "Allow casts from void *. Casts to and from void * are used for abstractions."
+        },
```

Comment:
After offline discussion with both @gordonwang0 and @gkwicker , disabling this rule seemed to be the way forward, for the reason stated.

Reply:
Suggest reword to "Allow casts from void *. Contexts are passed as void * and must be cast to the correct data type before use."